Utilities/CreateVault/create_vault.sh

```diff
@@ -17,54 +17,80 @@ if [ ! -d "${OCPath}" ]; then
   exit 1
 fi
 
-if [ ! -x /usr/bin/find ] || [ ! -x /bin/rm ] || [ ! -x /usr/bin/sed ] || [ ! -x /usr/bin/xxd ]; then
+if [ ! -x /usr/bin/env ] || [ ! -x /usr/bin/find ] || [ ! -x /bin/rm ] || [ ! -x /usr/bin/sed ] || [ ! -x /usr/bin/openssl ] || [ ! -x /usr/bin/awk ] || [ ! -x /usr/bin/sort ] || [ ! -x /usr/bin/xxd ]; then
   echo "Unix environment is broken!"
   exit 1
 fi
 
-if [ ! -x /usr/libexec/PlistBuddy ]; then
-  echo "PlistBuddy is missing!"
-  exit 1
-fi
-
-if [ ! -x /usr/bin/shasum ]; then
-  echo "shasum is missing!"
-  exit 1
-fi
-
 abort() {
   /bin/rm -rf vault.plist vault.sig /tmp/vault_hash
   echo "Fatal error: ${1}!"
   exit 1
 }
 
+# plist output functions so we don't need PlistBuddy
+write_header() {
+  cat <<EOF > "$1"
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Files</key>
+	<dict>
+EOF
+}
+
+write_file_name_and_hash() {
+  {
```

Comment:
Is this pair of braces really needed?

Reply:
I actually didn't know that was possible, the { cmd1; cmd2 } >> thing. Shellcheck suggested it all, pretty neat!